trackio/__init__.py

```diff
@@ -34,6 +34,8 @@ def init(
     project: str,
     name: str | None = None,
     space_id: str | None = None,
+    persistent_dataset: str | None = None,
+    persistent_dataset_dir: str | None = None,
```

Comment:
Do we need the `persistent_dataset_dir`? For simplicity for the user, can we exclude this and just choose a reasonable default path?